src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/service/TokenCertificateServiceTest.java

```diff
@@ -52,9 +53,12 @@
 import java.util.HashMap;
 import java.util.HashSet;
 
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.fail;
 import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doAnswer;
 import static org.mockito.Mockito.when;
+import static org.niis.xroad.restapi.service.TokenCertificateService.CERT_NOT_FOUND_FAULT_CODE;
 
 /**
  * Test TokenCertificateService
```

Comment:
This test does not mock all `TokenCertificateService` dependencies, so it is a bit integration test-y. I guess it should either a) be named `TokenCertificateServiceIntegrationTest` or b) add mocks for rest of the dependencies, such as GlobalConfFacade.

The problem with mocking only some of the dependencies is that it is easy to unknowingly use real dependencies and for example have GlobalConfFacade read a local /etc/xroad/globalconf in local dev env, and then have the tests fail on jenkins.

But this is a pre-existing problem and not related to this specific task.